example/cumulus-tf/main.tf

```diff
@@ -216,6 +215,15 @@ module "thin_egress_app" {
   vpc_subnet_ids             = var.lambda_subnet_ids
 }
 
+resource "aws_cloudwatch_log_subscription_filter" "egress_api_gateway_log_subscription_filter" {
+  count           = (var.log_api_gateway_to_cloudwatch != null && var.log_destination_arn != null) ? 1 : 0
```

Comment:
I modified count to include a check for `log_destination_arn` but I am not sure if there is a more elegant way to have this resource check on its dependent variables  (`log_api_gateway_to_cloudwatch` and `log_destination_arn`).

Reply:
No, I think this is fine

Reply:
However, since the default for `log_api_gateway_to_cloudwatch` is `false` and it has a type of `boolean`, the first check can just be`var.log_api_gateway_to_cloudwatch && ...`